daemon/health.go

```diff
@@ -92,7 +92,7 @@ func (d *Daemon) initHealth() {
 				d.cleanupHealthEndpoint()
 				return err
 			},
-			RunInterval: 30 * time.Second,
+			RunInterval: 60 * time.Second,
```

Comment:
Just a thought, but if the higher interval is needed just during the bootstrap phase, perhaps we could set it back to 30s after the first successful run. Not sure whether it's possible with the current controller implementation though.